app/views/provider_mailer/account_created.text.erb

```diff
@@ -0,0 +1,13 @@
+Dear <%= @provider_user.full_name %>,
+
+You've created an account with DfE Sign-in.
```

Comment:
This email needs to work in two contexts: either the user has just been invited to Sign-in and they need to open that email before continuing, or they're already set up on Sign-in and they can log in to Apply straight away (we don't know which state the user is in). How about:

```
Your account has been created on Manage teacher training applications.

You will need a DfE Sign-in account to use the service. If you already have a DfE Sign-in account you can proceed to the sign in page for Manage teacher training applications (text is a link). If you don't have a DfE Sign-in account we have sent you an email inviting you to create one.
```

Any ideas @laurahermionetennant92?

Reply:
@laurahermionetennant92 has added new copy to the Trello card